bom/openhab-addons/pom.xml

```
@@ -321,6 +321,11 @@
       <artifactId>org.openhab.binding.dmx</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+    <groupId>org.openhab.addons.bundles</groupId>
+      <artifactId>org.openhab.binding.ddominoswiss</artifactId>
```

This typo may cause your PR build to fail.

```suggestion
      <artifactId>org.openhab.binding.dominoswiss</artifactId>
```